public/scripts/controllers/booksController.js

```diff
@@ -0,0 +1,16 @@
+angular
+  .module("booksApp")
+  .controller("BooksController", BooksController);
+
+BooksController.$inject = ['$http', '$routeParams'];
+function BooksController(   $http,   $routeParams) {
```

Comment:
This file is using single-space indentation; most of the rest of your files use two-space indentation. Consistency would be nice, and a single space is not enough.